### DISCUSSION TITLE
Github generated weird link for github page

### DISCUSSION THREAD

**Discussion by MarkSuguitan-10** [community/community#23664] [Other Feature Feedback, Questions, & Ideas]
<p>Hi Sirs/Madam,</p>
<p>I am starting my portfolio website using GitHub pages.<br>
However the link generated by github seems redundant.<br>
Link Generated -&gt; <a href="https://marksuguitan-10.github.io/matsuguitan.github.io/" rel="nofollow noopener">https://marksuguitan-10.github.io/matsuguitan.github.io/</a><br>
But in the example in github pages it should be -&gt; <a href="http://username.github.io" rel="nofollow noopener">username.github.io</a>.<br>
Is there some mistake that I made with the creation of repo or pages?</p>

**Comment by SethClydesdale** [comment, marked as answer]
<p>Hello <a class="mention" href="/marksuguitan-10">@MarkSuguitan-10</a>, welcome to the GitHub Support Community!</p>
<p>The username for your GitHub page needs to be the same as your GitHub username.</p>
<p>Rename the repo from “<a href="http://matsuguitan.github.io" rel="nofollow noopener">matsuguitan.github.io</a>” to “<a href="http://marksuguitan-10.github.io" rel="nofollow noopener">marksuguitan-10.github.io</a>” and it should then work properly.</p>

**Comment by MarkSuguitan-10** [comment]
<p><a class="mention" href="/SethClydesdale">@SethClydesdale</a> Thank you very much. That really solved the problem. I hope they mentioned that in the get started page. :grinning:</p>

**Comment by RGGH** [comment]
<p>Thankyou, this really helped me - it would have been great if it had been pinned at top, or made more prominent.<br>
Many thanks again</p>
<p>I.</p>
